spaces/domains.json

```
@@ -229,4 +229,6 @@
   "vote.flashstake.io": "flashcommunity.eth",
   "vote.fenixtoken.net": "fenixtoken.eth",
   "vote.liquifi.org": "liquifi.eth"
+  "vote.honeydao.com": "honeydao.eth"
+}
```

A comma is missing at the end of the line above your sub-domain. Please also delete extra quote.
```suggestion
  "vote.liquifi.org": "liquifi.eth",
  "vote.honeydao.com": "honeydao.eth"
```